### ISSUE TITLE
Datastore getting corrupted on workers during reload

### ISSUE THREAD

**Issue by else** [containous/traefik#2328]
### Do you want to request a *feature* or report a *bug*?

bug

### What did you do?

I am using traefik with consul and consul_catalog and acme backend. Fresh startup (ie. with empty /traefik in consul) sometimes fails with: 

```
Error in watch datastore: invalid character 'c' after top-level value, retrying in 3.626249661s
```
The reason for this error is unclear to me, ie. I am not able to understand where this error is coming from. When this error shows up however no frontends are being served. Clearing the consul kv and restarting traefik sometimes helps.

### What did you expect to see?

No error, served frontends.

### Output of `traefik version`: (_What version of Traefik are you using?_)

1.4.0

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```
          "--debug",
          "--loglevel=DEBUG",
          "--defaultentrypoints=http,https",
          "--entrypoints=Name:http Address::80",
          "--entrypoints=Name:https Address::443 TLS",
          "--web.readonly",
          "--consul",
          "--consul.trace=true",
          "--consulcatalog",
          "--consulcatalog.exposedbydefault=false",
          "--consulcatalog.trace=true",
          "--consulcatalog.prefix=traefik",
          "--consulcatalog.watch",
          "--acme",
          "--acme.acmelogging",
          "--acme.entrypoint=https",
          "--acme.email=noc@foo.bar",
          "--acme.onhostrule",
          "--acme.storage=traefik/acme/account"
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

https://gist.github.com/else/9af633551036c5372a4112fe93aa8f33

**Comment by else** [comment]
Analyzed this further and logged the contents of the datastore meta: https://gist.github.com/else/c520f06d0252dfdb3ec6cb0a8edcc7bc

Somehow the contents is getting garbled.

**Comment by nmengin** [comment]
The problem seems to be linked to #2153

**Comment by else** [comment]
Indeed, looks like #2153 patched the wrong function (`Load` instead of `reload`).

**Comment by traefiker** [comment]
Closed by #2340.
